sys/include/random.h

```diff
@@ -27,6 +27,7 @@
 #define RANDOM_H
 
 #include <inttypes.h>
```

Comment:
not really this PR's fault but this should be stdint.h instead. stdint provides the uintxx_t which I assume is the reason for this header. inttypes in addition provides the PRIx32 etc printf format macros as well as some more stuff which is not used by this API header. (You don't have to address this comment in this PR)